pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["setuptools >= 64", "setuptools-scm >= 6.4"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling", "hatch-vcs"]
```

Comment:
Is this the recommended manner, as in they promise the api remains stable forever?

Reply:
I don't know if they're promising anything, but that's the recommended usage from the packaging guide.

Reply:
Yes, the behavior of the existing build options will never change. For reproducible builds you'd want to pin though.